src/tests/multihost/alltests/test_ldap_password_policy.py

```diff
@@ -0,0 +1,236 @@
+""" Automation of password Policy suite
+
+"""
+from __future__ import print_function
+import time
+import pytest
+import ldap
+import os
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from constants import ds_instance_name
+
+
+def find_logs(multihost, log_name, string_name):
+    "This function will find strings in a log file"
+    log_str = multihost.client[0].get_file_contents(log_name).decode('utf-8')
+    assert string_name in log_str
+
+
+def auth_success(client_hostname, user, user_pass):
```

Comment:
I am adding this module because this is more stable module currently present in our test framework  

Check this PR :  https://github.com/SSSD/sssd/pull/6431/files

Reply:
Then it might be a good idea to put this somewhere as a shared code that can be reused.
utils.py looks like the right place to me.
The name should imply that the authentication is via ssh and password.
Something auth_client_ssh_password might be a descriptive name.
The function should catch the exception that it can raise and return boolean result if auth succeeded/failed.
That way we can use it in assert and use it to check if login succeeded (we want/expect it to pass) /failed (we want/expect it to fail).

Having function with "success" or "failure"  is something that was done due to restraint/bash limitations and we want to have the functions more universal to support a better reuse.